lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -0,0 +1,57 @@
+import json
+import http.client
+import os 
+import boto3
+import pprint
+import urllib.parse
+
+def lambda_handler(event, context):
+    connection = http.client.HTTPSConnection(os.environ['vetext_api_endpoint_domain'])
+    headers = {'Content-type': 'application/json', 'Authorization': os.environ['vetext_api_endpoint_auth']}
+    
+    pp = pprint.PrettyPrinter(indent=4)
```

Comment:
You can use the shortcut function instead of instantiating the class: https://docs.python.org/3/library/pprint.html#pprint.pprint